Source/WebGPU/WebGPU/Device.h

```diff
@@ -281,6 +281,7 @@ class Device : public WGPUDeviceImpl, public ThreadSafeRefCountedAndCanMakeThrea
     NSMapTable<id<MTLCommandBuffer>, id<MTLCounterSampleBuffer>>* m_sampleCounterBuffers;
     NSMapTable<id<MTLCommandBuffer>, NSMutableArray<id<MTLBuffer>>*>* m_resolvedSampleCounterBuffers;
     id<MTLSharedEvent> m_resolveTimestampsSharedEvent { nil };
+    uint64_t m_commandEncoderId { 0 };
```

Comment:
At some point this will overflow, but even creating one encoder every nanosecond, it would take nearly 585 years to overflow, so I think this is ok